drivers/tee/tee_core.c

```diff
@@ -111,6 +126,162 @@ static int tee_release(struct inode *inode, struct file *filp)
 	return 0;
 }
 
+static const char *tee_session_get_application_id(void)
+{
```

Comment:
These two functions are unused in this patch, should be moved into the "RFC" patch.

Reply:
Ash... they were there at one point of time. will fix it. Sorry about it.